gateway/conf/nginx.conf.liquid

```diff
@@ -41,6 +41,9 @@ http {
   lua_package_path "{{ lua_path | default: package.path }}";
   lua_package_cpath "{{ lua_cpath | default: package.cpath }}";
 
+  underscores_in_headers on;
```

Comment:
Is this needed if `ignore_invalid_headers` is set to `off`?
From the docs:

```
 When the use of underscores is disabled, request header fields whose names contain underscores are marked as invalid and become subject to the ignore_invalid_headers directive.
```

Reply:
Good point. Lets try.

Reply:
Indeed. Fixed.